pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -48,6 +48,11 @@ stdenv.mkDerivation rec {
       ./rtcwake-search-PATH-for-shutdown.patch
       # https://github.com/util-linux/util-linux/pull/3013
       ./fix-darwin-build.patch
+      # fixes https://github.com/util-linux/util-linux/issues/3474
+      (builtins.fetchurl {
```

Comment:
We don't use builtins.fetchurl inside Nixpkgs, and fetching patches from GitHub commits without going through fetchpatch(2) is non-reproducible in any case.  We would need to include the patch file in Nixpkgs like the other patches here.

Reply:
Will fix :slightly_smiling_face:

Reply:
The current documentation is at https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches, with some more specific language currently being discussed in #401863.